src/ng/animate.js

```diff
@@ -369,6 +369,19 @@ var $AnimateProvider = ['$provide', /** @this */ function($provide) {
        * );
        * ```
        *
+       * <div class="alert alert-warning">
+       * **Note**: Generally, the events that are fired correspond 1:1 to `$animate` method names,
+       * e.g. {@link ng.$animate#addClass addClass} will fire `addClass`, and {@link ng.ngClass}
+       * will fire `addClass` if classes are added and `removeClass` if a classes are removed.
+       * However, there are two exceptions:
+       * <ul>
```

Comment:
Doesn't markdown work here?

Reply:
Not lists. They are transformed to italics instead. Maybe it's a whitespace / newline issue

Reply:
Pretty sure it should work with markdown. It might be that you need blank lines between the blocks.

Reply:
I tried a few things, including blank lines, but it didn't work